tenancy-api-mapping/VERSION

```diff
@@ -1 +1 @@
-1.6.0
+1.5.5
```

Comment:
why are we going back?

Reply:
This was a mistake, fixed in [095ae67](https://github.com/open-edge-platform/orch-utils/pull/135/commits/095ae6797182d8d003e0a2da11d1757bc1b8d499)